### PR TITLE
Feature create works & about me sections

### PR DESCRIPTION
**Pull request by Roathy**
## Description

Create works section and about me section for the mobile version of the project.

## Checklist:
- [x]  No linter errors.
- [x] Follow github flow.
- [x] Descriptive README.md file.
- [x] Mobile version for works section.
- [x] Mobile version for about me section.
- [x] Use of Flexbox and Grid combined. 
- [x] Use ul and li in categories and skills.

### CODE REVIEW

**Review by Efiamotu-1 — COMMENTED** [review]
Hi @Roathy , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that cannot be reviewed due to request when submitting the pr, 
- [ ] However, to help you solve your issue with your previous project, i have reached you on slack so open the app and lets connect.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,275 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css"
+        integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw=="
```
Comment (bobb-Rob) [inline index.html]: How the README should be
image


Your README 
image

**Review by bobb-Rob — COMMENTED** [review]
Hi @Roathy, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that is NOT using the correct README file as some sections are not visible. Kindly use the recommended README and note that all sections, beginning from the table of content to Licensing should NOT be left out. 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: index.html [hunk 2]
```diff
@@ -0,0 +1,275 @@
...
+                        <li>
+                            <div class="box-background">
+                                <p>JavaScript</p>
+                            </div>
+                        </li>
+                    </ul>
+                </div>
+                <div class="normal-button">
+                    <p>See project</p>
+                </div>
```
Comment (didierganthier) [inline index.html]: Well done on the project 💯 but the button (e.g., "See project") should be a link `a` or `button` tag, and follow the button styling: Active, Hover, and Pressed (see design) 👇🏽 

Screenshot 2023-04-27 at 23 29 45

File: index.html [hunk 3]
```diff
@@ -0,0 +1,275 @@
...
+                    <li> <a href="#"><i class="fa-brands fa-github"></i></a></li>
+                    <li> <a href="#"><i class="fa-brands fa-linkedin-in"></i></a></li>
+                    <li> <a href="#"><i class="fa-regular fa-hand-peace fa-flip-horizontal"></i></a></li>
+                    <li> <a href="#"><i class="fa-brands fa-twitter"></i></a></li>
+                    <li> <a href="#"><i class="fa-sharp fa-solid fa-m"></i></a></li>
+                </ul>
+            </div>
+            <div class="normal-button">
+                <p>Get my resume</p>
+            </div>
```
Comment (didierganthier) [inline index.html]: Well done on the project 💯 but the button (e.g., "Get my resume") should be a link `a` or `button` tag, and follow the button styling: Active, Hover and Pressed (see design) 👇🏽 

Screenshot 2023-04-27 at 23 29 45

**Review by didierganthier — CHANGES_REQUESTED** [review]
Hi @Roathy ,

🎉 Good job so far! You're making great progress on this project, but there are still a few areas that need attention before we can move on to the next step.



## Highlights
- Descriptive PR summary ✔️ 
- Nice UI design ✔️ 
- Correct GitHub flow ✔️ 

## Required Changes 🛠
- Please review the comments under the review for specific changes that still need to be made.

## Optional Suggestions 💡
- N/A

🚀 Keep pushing forward, you're almost there! If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread and don't forget to tag me so I'll receive the notification.

**Please note**
- To avoid confusion and delays, please do not open a new pull request for re-reviews. Use the same pull request submitted for the first review, unless otherwise instructed.

------
As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). 

Cheers and happy coding! 👏👏👏

**Review by AdnanOlarmmi — APPROVED** [review]
Hi @Roathy,

This is Adnan Is-haq, your beloved code reviewer, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Approved office man

## Highlights
- Fixed reviewer's suggestion.
- Working app.

## Optional suggestions
_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**